corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap3/email-request.js

```diff
@@ -89,11 +95,15 @@ hqDefine('hqwebapp/js/bootstrap3/email-request', [
             return regex.test(email);
         }
 
+        function isDimagiEmail(email) {
+            return email.includes('@dimagi.com');
```

Comment:
nit: use `endsWith` instead of `includes` for more robust check

Reply:
That's a good idea. The `isEmail` check should already ensure that its in the correct format, but it can't hurt to add `endsWith` here as well.

Addressed in ([fd4133a](https://github.com/dimagi/commcare-hq/pull/34768/commits/fd4133a6b446bf7a00f7e398f73d39f01dc8abf1)).